tests/cuda/test_cuda_containers_kernels.cu

```diff
@@ -33,6 +33,11 @@ void linearTransformKernel( std::size_t size, const int* input, int* output ) {
 void linearTransform( std::size_t size, const int* input, int* output ) {
    linearTransformKernel<<< 1, size >>>( size, input, output );
 
-   VECMEM_CUDA_ERROR_CHECK( cudaGetLastError() );
-   VECMEM_CUDA_ERROR_CHECK( cudaDeviceSynchronize() );
+   if (cudaGetLastError() != cudaSuccess) {
+      throw std::runtime_error("Failure!");
+   }
+
+   if (cudaDeviceSynchronize() != cudaSuccess) {
+      throw std::runtime_error("Failure!");
+   }
```

Comment:
:confused: Why didn't you use the (now) private `cuda_error_handling.hpp` header in the end? I would still go that route with these unit tests... Unless you want to introduce a separate checker macro for the tests. But I think that's not necessary.

Later on we will hopefully have a lot of tests with a lot of return codes to check. We shouldn't start checking them one by one like this.

Reply:
I'm extremely hesitant to use relative include paths across the test-source border, but we can always address that problem when we run into it later when we have more tests. So I'll include the macro by relative path for now.